handler/error.go

```diff
@@ -37,12 +37,15 @@ func (e *Error) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	}
 
 	for _, kind := range err.Kinds() {
+		if kind == "access_control" {
```

Comment:
why override a local var ?
A better place would be the fallback wildcard lookup at line 61. Which means the Interface lookup on the endpoint handler may turn into a method.

Reply:
refactored